client/signup/config/flows.js

```diff
@@ -223,13 +223,22 @@ const flows = {
 	},
 
 	'domain-first': {
```

Comment:
It looks like this entry can be now safely removed as intention is to load this flow only when config flag is set to `true`.